services/server/server.go

```diff
@@ -65,7 +66,11 @@ type Server struct {
 	platformService      platformService
 	awsInstanceChecker   *AWSInstanceChecker
 	grafanaClient        grafanaClient
-	l                    *logrus.Entry
+	RulesService         *ia.RulesService
+	AlertsService        *ia.AlertsService
```

Comment:
`AlertsService` and `TemplatesService` is not used in `Server`, let's remove it from here